templates/person/show.json.twig

```diff
@@ -0,0 +1,71 @@
+{
+    "@context": "https://schema.org",
+    "@graph": [
+        {
+            "@type": "Person"
+            {% if person.firstname %},"givenName": "{{ person.firstName}}" {% endif %}
+            {% if person.lastName %},"familyName": "{{ person.lastName}}" {% endif %}
+            ,"gender": "{{ person.gender }}"
+            {% if (person.wikipediaUrl) or (person.viafUrl) %}
+            ,"sameAs": [
+                {% if person.wikipediaUrl %} "{{ person.wikipediaUrl}}"{% endif %}
+                {% if (person.wikipediaUrl) and (person.viafUrl) %} , {% endif %}
+                {% if person.viafUrl %} "{{ person.viafUrl}}" {% endif %}
+            ]
+            {% endif %}
+            {% if person.dob %},"birthDate": "{{person.dob}}" {% endif %}
+            {% if person.dod %}, "deathDate": "{{person.dod}}" {% endif %}
+            {% if person.cityOfBirth %}, "birthPlace": "{{person.cityOfBirth}}" {% endif %}
+            {% if person.cityOfDeath %}, "deathPlace": "{{person.cityOfDeath}}" {% endif %}
```

Comment:
All locations should be in the same format--i.e. as a locationCreated object with GeoCoordinates.